libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLBetween2277And3XSchemaAccessor.java

```diff
@@ -231,4 +233,54 @@ public List<DBTableColumn> listTableColumns(String schemaName, String tableName)
         }
         return super.listTableColumns(schemaName, tableName);
     }
+
+    @Override
+    public Map<String, List<DBTableColumn>> listBasicTableColumns(String schemaName) {
+        String sql = sqlMapper.getSql(Statements.LIST_BASIC_SCHEMA_TABLE_COLUMNS);
+        List<DBTableColumn> tableColumns = jdbcOperations.query(sql, new Object[] {schemaName},
+                listBasicTableColumnRowMapper());
+        return tableColumns.stream().collect(Collectors.groupingBy(DBTableColumn::getTableName));
+    }
+
+    @Override
+    public Map<String, List<DBTableColumn>> listBasicViewColumns(String schemaName) {
+        MySQLSqlBuilder sb = new MySQLSqlBuilder();
+        sb.append("select table_name from information_schema.views where table_schema=");
+        sb.value(schemaName);
+        List<String> viewNames = jdbcOperations.query(sb.toString(), (rs, rowNum) -> rs.getString("table_name"));
+        List<DBTableColumn> columns = new ArrayList<>();
+        for (String viewName : viewNames) {
+            columns.addAll(fillViewColumnInfoByDesc(schemaName, viewName));
```

Comment:
performace risk , avoid dict view query in looop

Reply:
This is a work around way, more details:
<img width="823" alt="image" src="https://github.com/oceanbase/odc/assets/84772536/1fed02db-e3ee-4efe-83fb-2c301526ac69">